modules/RTC/JitsiLocalTrack.js

```diff
@@ -559,13 +559,18 @@ export default class JitsiLocalTrack extends JitsiTrack {
      * NOTE: used only for audio tracks to detect audio issues.
      */
     _setByteSent(tpc, bytesSent) {
```

Comment:
does this method get called multiple times?  then we'd call setTimeout each time, no? is that what we want?

Reply:
It is called multiple times but the setTimeout will be called only ones because of the _testByteSent property.

Reply:
`this._testByteSent` will be true any time it's non-zero though?

Reply:
It is true until we execute `setTimeout` and after that we are setting it to false.

Reply:
oops, i got confused between the two variables.  i see now.